sdk/src/accounts/README_WebSocketAccountSubscriberV2.md

```diff
@@ -31,6 +31,46 @@ await subscriber.subscribe((data) => {
 await subscriber.unsubscribe();
 ```
 
+### Polling Instead of Resubscribing
+
+For accounts that rarely update (like long-tail markets), you can use polling instead of resubscribing to reduce resource usage:
+
+```typescript
+const resubOpts = {
+	resubTimeoutMs: 30000, // 30 seconds
+	logResubMessages: true,
+	usePollingInsteadOfResub: true, // Enable polling mode
+	pollingIntervalMs: 30000, // Poll every 30 seconds (optional, defaults to 30000)
+};
+
+const subscriber = new WebSocketAccountSubscriberV2(
+	'perpMarket', // account name
+	program,
+	marketPublicKey,
+	undefined, // decodeBuffer
+	resubOpts
+);
+```
+
+**How it works:**
+1. Initially subscribes to WebSocket updates
+2. If no WebSocket data is received for `resubTimeoutMs` (30s), switches to polling mode if `usePollingInsteadOfResub` is specified true, else just resubscribes(unsub, sub).
+3. Polls every `pollingIntervalMs` (30s) to check for updates by:
```

Comment:
probably good to mention something like "separately polls", just because the context above this point talks about polling replacing WS resub, but this polling happens even with websocket subscribed